commit 5b1949a3c6dd168a85b899cba759b3efa6d126fa
Author: QbProg <tholag@gmail.com>
Date:   Thu May 12 20:53:28 2011 +0200

    [warning-fix][linker-warnings]
    
    Fixes warnings like : local symbol 'name' imported
    
    in this case there was missing a case in the ifdef for __Standard_API, so a bunch of HashCode functions were wrongly imported.

Index: oce/inc/Standard_Macro.hxx
===================================================================
--- oce.orig/inc/Standard_Macro.hxx
+++ oce/inc/Standard_Macro.hxx
@@ -117,7 +117,7 @@
 
 # ifndef __Standard_API
 //#  ifdef WNT
-#   if !defined(WNT) || defined(__Standard_DLL) || defined(__FSD_DLL) || defined(__MMgt_DLL) || defined(__OSD_DLL) || defined(__Plugin_DLL) || defined(__Quantity_DLL) || defined(__Resource_DLL) || defined(__SortTools_DLL) || defined(__StdFail_DLL) || defined(__Storage_DLL) || defined(__TColStd_DLL) || defined(__TCollection_DLL) || defined(__TShort_DLL) || defined(__Units_DLL) || defined(__UnitsAPI_DLL) || defined(__Dico_DLL)
+#   if !defined(WNT) || defined(__Standard_DLL) || defined(__FSD_DLL) || defined(__MMgt_DLL) || defined(__OSD_DLL) || defined(__Plugin_DLL) || defined(__Quantity_DLL) || defined(__Resource_DLL) || defined(__SortTools_DLL) || defined(__StdFail_DLL) || defined(__Storage_DLL) || defined(__TColStd_DLL) || defined(__TCollection_DLL) || defined(__TShort_DLL) || defined(__Units_DLL) || defined(__UnitsAPI_DLL) || defined(__Dico_DLL) || defined(__Message_DLL)
 #    define __Standard_API Standard_EXPORT
 #    define __Standard_APIEXTERN Standard_EXPORTEXTERN
 #   else
